### PR TITLE
Remove interactivity utils

### PR DESCRIPTION
**Pull request by kullJul**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-utils-chartutils",
-  "version": "6.0.4",
+  "version": "6.0.5",
```
Comment (AleksSavelev) [inline package.json:3]: Let's increase the version to 7.0.0

File: src/label/labelLayout.ts [hunk 2]
```diff
@@ -361,9 +359,13 @@ export interface Label extends interactivitySelectionService.SelectableDataPoint
     backgroundColor?: string;
 
     backgroundTransparency?: number;
+
+    selected: boolean;
+
+    identity: ISelectionId;
```
Comment (AleksSavelev) [inline src/label/labelLayout.ts:365]: Let's create an interface as it was in interactivityutils

Comment (AleksSavelev) [reply src/label/labelLayout.ts:365]: Also we will be able to use this interface in `src/dataLabel/dataLabelUtils.ts`

File: package.json (hunk 1 above)
Comment (kullJul) [reply package.json]: fixed 1aad98d1d3240cd7650254b1610f2b4f007af271

File: src/label/labelLayout.ts (hunk 2 above)
Comment (kullJul) [reply src/label/labelLayout.ts]: done e9a0ea20962684b1aab1ae52e72eb409c1f1bf57

**Review by AleksSavelev — APPROVED** [review]

**Review by AleksSavelev — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-utils-chartutils",
-  "version": "6.0.4",
+  "version": "7.0.0",
```
Comment (AleksSavelev) [inline package.json:3]: Let's change it to 7.0.0-beta.1

**Review by AleksSavelev — APPROVED** [review]

**Pull request merged**